client/src/api.ts

```diff
@@ -831,7 +831,7 @@ export async function uploadThumbnail(
       "Content-Type": "multipart/form-data",
     },
   });
-  return getDataFromAxiosResponse(result, []);
+  return getDataFromAxiosResponse(result, ["thumbnail"]);
```

Comment:
this probably needs to be `['data', 'thumbnail']` (because this request isn't actually graphql)?

Reply:
I actually as that and couldn't figure why it wasn't working but changing it to just "thumbnail" fixed it somehow
the actual response has the shape
data.data.thumbnail

Reply:
The json in the response is just 

```json
{ "data": { "thumbnail": "" } }
```

...the first `data` property you're referring to is a `AxiosResponse` object